compatibility.json

```diff
@@ -1,5 +1,6 @@
 {
   "rpcChainVMProtocolVersion": {
+    "v0.4.4": 19,
```

Comment:
should we add this to README as well? if we feel like this is a duplicate with README I think we can move readme's compatibility chart here and add avago versions here

Reply:
Hmm I'm not sure we should add this to the README until it's released since we may change it before actually doing the release. Maybe we shouldn't have it here either until the release, but that would change our ability to update `plugin/evm/version.go` at the start of the release cycle as opposed to the end. I thought about removing it from the README entirely, but think that it is more user friendly to keep the documentation there in the normal format rather than refer people to a JSON file.

cc @dasconnor does including an unreleased version in this JSON file break anything for the CLI? The intention would be to merge this to the master branch at the start of the release cycle and if a PR needs to update the avalanchego dependency and the RPCChainVMProtocol verison that same PR would need to update this JSON file, but it may have unreleased and unfinalized information for the "latest" until then.